src/styles/main.scss

```diff
@@ -1,3 +1,78 @@
+$cell-size: 100px;
+$gap-space: 1px;
+$calendar-size: $cell-size * 7 + $gap-space * 6;
+$cell-color: #eee;
+$cell-hover-color: pink;
+$cell-border-color: black;
+
+html {
+  font-family: "Arial", "Helvetica", sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+.calendar {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__month {
+    display: flex;
+    width: $calendar-size;
+    flex-wrap: wrap;
+    gap: $gap-space;
+  }
+
+  @for $index from 1 through 31 {
+
+    &__day:nth-child(#{$index}) {
+      display: flex;
+      align-items: center;
```

Comment:
you can write just  &__day instead of &__day:nth-child(#{$index})

Reply:
Agree